File: Examples/Input/prueba_04_entrada_salida.cs
using System;

class Conversor
{
    public static void Main(string[] args)
    {
        Console.WriteLine("=== Conversor de Unidades ===");

        Console.WriteLine("Ingresa los centimetros:");
        int cm = int.Parse(Console.ReadLine());

        int metros  = cm / 100;
        int sobrante = cm % 100;

        Console.Write("Metros enteros: ");
        Console.WriteLine(metros);
        Console.Write("Centimetros sobrantes: ");
        Console.WriteLine(sobrante);

        Console.WriteLine("Conversion completada.");
    }
}

File: Examples/Output/prueba_04_entrada_salida.java
import java.util.Scanner;

class Conversor {
    private static Scanner scanner = new Scanner(System.in);
    public static void main(String[] args) {
        System.out.println("=== Conversor de Unidades ===");
        System.out.println("Ingresa los centimetros:");
        int cm = Integer.parseInt(scanner.nextLine());
        int metros = (cm / 100);
        int sobrante = (cm % 100);
        System.out.print("Metros enteros: ");
        System.out.println(metros);
        System.out.print("Centimetros sobrantes: ");
        System.out.println(sobrante);
        System.out.println("Conversion completada.");
    }
}

